packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -422,6 +422,7 @@ contract Connext is
     (address _transactingAssetId, uint256 _amount) = AssetLogic.transferAssetToContract(
       _args.transactingAssetId,
       _args.amount,
+      _args.relayerFee,
```

Comment:
should assert that the relayer fee can't be 0

Reply:
Updated here https://github.com/connext/nxtp/pull/955/commits/01b71b34666cce4f65299932f3343fa83f9aac18